app/app.py

```diff
@@ -101,10 +119,29 @@ def add_rating():
     return render_template("form.html", title="Add Visit", restaurants=restaurant_in_database)
 
 
-@app.route("/admin_panel")
-@roles_required(["Admin"])
+@app.route("/admin_panel", methods=["GET", "POST"])
+@roles_required(["Admin", "User"])
```

Comment:
czyli na razie wpuszczamy też userów?

Reply:
Stąd moje pytanie na standupie:
1) Skoro wpuszczamy tylko Adminów, to jak mianujemy Adminów? Nikt nie będzie mógł wejść na panel admiński dopóki jakiś admin nie pojawi się:
    a) dopinamy się do bazy na masterze i ręcznie wklepujemy mu admin użytkownika
    b) na razie mamy to gdzieś i robimy pełną samowolkę, dopóki nie ma userów można szaleć.

Reply:
ok, zostawmy tak na razie 🤙